semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/span/SpanSemanticConventionUtils.java

```diff
@@ -31,4 +33,40 @@
     }
     return Optional.of(host);
   }
+
+  /**
+   * @param attributeValueMap map of span data and attribute value
+   * @return URI based on OTel format
+   */
+  public static Optional<String> getURIForOtelFormat(Map<String, AttributeValue> attributeValueMap) {
```

Comment:
nit: I wonder why we use `URI` here and `Url` in the previous file. Not sure what is idiomatic in java.

Reply:
I will raise a separate pr to make it consistent at all places.